glusterd2/commands/volumes/options.go

```diff
@@ -0,0 +1,13 @@
+package volumecommands
+
+type option struct {
+	OptionName string
+	OnValue    string
+	OffValue   string
+}
+
+// GroupOptions maps from a profile name to a set of options
+var GroupOptions = map[string][]option{
```

Comment:
Does this mean we 'd need to have all the group profile options  of each profile dumped into this structure? If that's the case, then this is not flexible. Consider someone wants to create his/her own custom profile with different tunables. How can we achieve it?

Reply:
GroupOptions map is loaded once into the store during initialization and it is fetched from the store each time a group profile option is set/unset.
So if we allow the user to modify the in-store state of GroupOptions, that would allow the creation of custom profiles.